results2junit

```diff
@@ -242,7 +242,7 @@ sub dump_junit {
             $test_suite_name, $class_name;
         printf $out_fh " name=\"%s\"", $test_name;
         printf $out_fh " time=\"%d\"",  $s->{run_time};
-        if ($s->{status} eq 'GOOD') {
+        if ($s->{status} eq 'GOOD' || $s->{status} eq 'TEST_NA') {
             print $out_fh "/>\n";
```

Comment:
Hm, but then are we really going to want to see test failures each time because CentOS is running a rhel-only plugin, or docker-1.10 is running a 1.12-only test? Are there different levels of messages other than <error> and <failure> ?